specification/attributes/key_value_format.md

```diff
@@ -0,0 +1,32 @@
+# Key-Value Format
+
+Columns that provide Key-Value information are often used in place of separate columns for enumerating data which would be inherently sparse and/or without predetermined keys. This consolidates related information and provides more consistency in the schema. Key-value pairs are also referred to as name-value pairs, attribute-value pairs, or field-value pairs.
+
+All key-value related columns defined in the FOCUS specification MUST follow the key-value formatting requirements listed below. These columns SHALL NOT contain nested elements.
+
+## Attribute ID
+
+KeyValueFormat
+
+## Attribute Name
+
+Key-Value Format
+
+## Description
+
+Rules and formatting requirements for columns appearing in billing data which convey data as key-value pairs.
+
+## Requirements
+
+* Key-Value Format columns MUST contain a serialized JSON string, consistent with the ECMA-404 definition of an object.
```

Comment:
Do we want to have an opinion on case sensitivity? As it stands this is permitting "keynull" and "KeYNuLl". Some CSP's have case sensitive tags, some CSP's don't.

Reply:
never mind. it is probably use case dependent (e.g. this is a discussion better left for `Tags`)